src/status_im/ui/screens/events.cljs

```diff
@@ -232,18 +232,29 @@
 
 (register-handler-db
   :initialize-account-db
-  (fn [db _]
-    (-> db
-        (assoc :current-chat-id console-chat-id)
-        (dissoc :transactions
-                :transactions-queue
-                :wallet
-                :contacts/new-identity))))
+  (fn [{:keys [accounts/accounts networks/networks network view-id navigation-stack
+               status-module-initialized? status-node-started?] :as db} [_ address]]
+    (let [network' (or network (get app-db :network))]
```

Comment:
A good candidate for `:or` inside `:keys` destructuring?

Reply:
yeah thanks didn't put enough thought into this I was just copying from the one above